### DISCUSSION TITLE
Добавление разных голосов

### DISCUSSION THREAD

**Discussion by Sur4n** [ilyhalight/voice-over-translation#680] [Ideas]
Хотел бы узнать, возможно ли добавление новых голосов? Просто стандартный звучит уж очень сухо, хотя приемлемо. Думаю это вполне осуществимо, ведь сколько уже существует различных голосов от нейронок.

**Comment by NullVerdict** [comment]
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
Нет. Озвучка происходит на серверах Яндекса и как-либо повлиять на нее нельзя.
#533 
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/c68fe274-c701-4a5c-aa0e-1616a15be7d3)
